### PR TITLE
Wishlist

### PR DESCRIPTION
**Pull request by anand-tiwari**

### CODE REVIEW

**Comment by anand-tiwari** [comment]
Can Preview UI  here - https://deploy-preview-4--amazing-almeida-c905b7.netlify.app/wishlist/index.html

File: wishlist/index.html [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+        </nav>
+
+        <div class="main-container">
+            <div class="wishlist__title">MY WISHLIST</div>
+            <div class="products">
+                <div class="product__item">
+                    <div class="card__container card__container_vertical">
+                        <div class="card__imagecontainer card__imagecontainer_vertical">
+                            <img src="../images/shoes1.webp">
+                            <span class="badge badge-trending">Trending</span>
```
Comment (talrejah02) [inline wishlist/index.html:42]: use alt attribute in img tags everywhere ,apart this you are good to go

**Review by talrejah02 — APPROVED** [review]

**Comment by psychoactiv** [comment]
Looks good to merge

**Review by tewarimohit — COMMENTED** [review]
code looks good to me...

**Pull request merged**